frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/vm-settings-tab/flavor.tsx

```diff
@@ -62,6 +62,7 @@ export const FlavorSelect: React.FC<FlavorProps> = React.memo(
       acc[getFlavor(tmp)] = getFlavorText({
         cpu: getCPU(selectVM(tmp)),
         memory: getMemory(selectVM(tmp)),
+        t,
```

Comment:
```suggestion
        flavor: getFlavor(tmp),
        t,
```

Reply:
https://github.com/openshift/console/pull/7836/files#diff-89381ef5e018743a88bb9882246151f255ec461c08bb4b8121fe44ca94c624bdR129-R131

should also be fixed to use:
``` js
{isDefault
  ? t('kubevirt-plugin~{{flavor}} (default)', { flavor: flavorDesc })
  : flavorDesc}
```